recipes/lzo/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "lzo" %}
+{% set version = "2.09" %}
+{% set version_major = "2" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "f294a7ced313063c057c504257f437c8335c41bfeed23531ee4e6a2b87bcb34c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.oberhumer.com/opensource/{{ name }}/download/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    # Patch imported from https://github.com/Microsoft/vcpkg/tree/ed8e7ca79dad5b7bcf716efe25d975571e8e8c05/ports/lzo
+    - do-not-declare-setargv.patch  # [win]
```

Comment:
> commits are just releases imports

Yeah maybe I should have said "mirror". Though they appear to attribute the right author.

> Giving it a try: https://github.com/nemequ/lzo/issues/1

Thanks @rolando. Bare minimum they can tell us where to send the patch.